packages/esm-patient-registration-app/src/patient-registration/section/extra-information/extra-information.component.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useContext, useEffect } from 'react';
+import styles from './../section.scss';
+import { useField } from 'formik';
+import { getField } from '../section-helper';
+import { PatientRegistrationContext } from '../../patient-registration-context';
+import { Input } from '../../input/basic-input/input/input.component';
+import { useTranslation } from 'react-i18next';
+import { useConfig } from '@openmrs/esm-framework';
+import camelCase from 'lodash-es/camelCase';
+
+export interface ExtraInformationSectionProps {
+  id: 'extraInformation';
+  fields: Array<any>;
+}
+
+export const ExtraInformationSection: React.FC<ExtraInformationSectionProps> = ({ fields }) => {
+  const [field, meta] = useField('attributes');
+  const { setFieldValue } = useContext(PatientRegistrationContext);
+  const { personAttributes } = useConfig();
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    if (!field.value && meta.touched) {
+      setFieldValue('additionalGivenName', '');
+      setFieldValue('additionalMiddleName', '');
+      setFieldValue('additionalFamilyName', '');
+    }
+  }, [field.value, meta.touched]);
+
+  return personAttributes?.length ? (
+    <section className={styles.formSection} aria-label="Extra Information Section">
+      {personAttributes
+        ?.filter((personAttribute) => personAttribute.type === 'coded')
```

Comment:
Are we sure we want to restrict this to `coded` attributes?

Reply:
Hi @ibacher! As per the discussion in the issue thread, it was decided. But yes, I too think it might be unnecessary now.

Reply:
Yeah, I suppose there is a problem if we want certain "extra-information" fields (like email of phone number) to show up in a particular place...

Reply:
Yeah, correct.